src/main/scala/cromwell/Main.scala

```diff
@@ -53,29 +39,33 @@ object Main extends App {
     )
 
     binding.workflow.calls.foreach { call =>
-      println(s"Call: $call")
```

Comment:
I'll merge the PR anyway... but all this can go away soon, it was just early dev testing stuff